src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Unit.Tests/CompilationHelper.cs

```diff
@@ -278,57 +319,9 @@ public class Location
             return CreateCompilation(source);
         }
 
-        public static Compilation CreateRepeatedLocationsWithResolutionCompilation()
-        {
-            string source = """
-                using System;
-                using System.Collections;
-                using System.Collections.Generic;
-                using System.Text.Json.Serialization;
-
-                [assembly: JsonSerializable(typeof(Fake.Location))]
```

Comment:
Removing test using a deprecated activation gesture that no longer compiles succesfully.